draft-ietf-tls-tls13.md

```diff
@@ -1589,8 +1593,9 @@ offered by the client in its ClientHello, with the exception of optionally the
 "cookie" (see {{cookie}}) extension.
 
 Upon receipt of a HelloRetryRequest, the client MUST verify that the extensions
-block is not empty and otherwise abort the handshake with a fatal
-"decode_error" alert.  Clients SHOULD also abort with "unexpected_message" in
+block is not empty and MUST abort the handshake with a
```

Comment:
This text says that a client that receives a HRR MUST abort the handshake.  (That should actually only happen if the extensions block is empty.)
Probably just "is not empty and abort the handshake with a "decode_error" alert if it is empty." would suffice.